gen/dcos-config.yaml

```diff
@@ -2206,7 +2206,10 @@ package:
       # Read metrics from DC/OS Net Prometheus endpoint.
       [[inputs.prometheus]]
         ## An array of urls to scrape metrics from.
-        urls = ["http://127.0.0.1:62080/v1/metrics/default"]
+        urls = [
+          "http://127.0.0.1:62080/v1/metrics/default"
+          "http://127.0.0.1:62080/v1/metrics/dns"
```

Comment:
I would like to add `dns_` prefix for all dns metrics in telegraf, i.e.

```
[[inputs.prometheus]]
  ## An array of urls to scrape metrics from.
  urls = ["http://127.0.0.1:62080/v1/metrics/dns"]
  ## Specify timeout duration for slower prometheus clients (default is 3s)
  response_timeout = "30s"
  name_prefix = "dns_"
  ## Apply DC/OS component name tag according to the documentation.
  [inputs.prometheus.tags]
    dcos-component-name = "DC/OS Net"
```

Reply:
Makes a lot of sense.

Reply:
Should we also rename the dcos-component-name to "DC/OS Net DNS"?

Reply:
No, we shouldn't. The component is still called "DC/OS Net". Maybe we should add one more tag in the future, but at the moment let's keep it simple. We always can change it. That's exactly why I suggested to have registry per subproject.

Reply:
Ok, I fixed the label back to "DC/OS Net".